app/src/main/res/raw/snowflake_params

```diff
@@ -0,0 +1 @@
+url=https://snowflake-broker.torproject.net.global.prod.fastly.net/ front=cdn.sstatic.net ice=stun:stun.l.google.com:19302,stun:stun.voip.blackberry.com:3478,stun:stun.altar.com.pl:3478,stun:stun.antisip.com:3478,stun:stun.bluesip.net:3478,stun:stun.dus.net:3478,stun:stun.epygi.com:3478,stun:stun.sonetel.com:3478,stun:stun.sonetel.net:3478,stun:stun.stunprotocol.org:3478,stun:stun.uls.co.za:3478,stun:stun.voipgate.com:3478,stun:stun.voys.nl:3478
```

Comment:
Sorry if I've got this backwards, but `raw` is using Fastly and `raw-v25` is using Azure. Older Android versions have a problem with Fastly, not Azure, so shouldn't that be the other way around?